kafka-persistence/src/main/scala/com/evolutiongaming/kafka/flow/kafkapersistence/package.scala

```diff
@@ -0,0 +1,52 @@
+package com.evolutiongaming.kafka.flow
+
+import cats.Parallel
+import cats.effect.{Concurrent, Resource, Timer}
+import com.evolutiongaming.catshelper.LogOf
+import com.evolutiongaming.kafka.flow.timer.TimersOf
+import com.evolutiongaming.kafka.journal.ConsRecord
+import com.evolutiongaming.skafka.{Offset, TopicPartition}
+import com.evolutiongaming.smetrics.MeasureDuration
+
+package object kafkapersistence {
+
+  implicit class PartitionFlowOfCompanionOps(val self: PartitionFlowOf.type) extends AnyVal {
+
+    /**
+      * Creates PartitionFlowOf which on partition assignment reads respective partition of "snapshot" (usually compacted)
+      * topic and eagerly recovers all the state from it.
+      */
+    def eagerRecoveryKafkaPersistence[F[_] : Concurrent : Timer : Parallel : MeasureDuration : LogOf, S](
```

Comment:
Why it should be an implicit extension method? Not saying it should not, just could not yet understand.

Reply:
My intention was to stress the fact that it's yet another constructor in `PartitionFlowOf` toolset and to provide consistent experience to client code writers, like `PartitionFlowOf.xxx` given the correct imports. Otherwise I'd need to come up with separate creatively named class or constructor object and as you rightly said, naming is hard:)